keyboards/handwired/colorlice/readme.md

```diff
@@ -0,0 +1,13 @@
+# COLORLICE
+
+A keyboard inspired by TGR Alice
+
+* Keyboard Maintainer: [marhalloweenvt](https://github.com/marhalloweenvt)
+* Hardware Supported: Colorlice PCB
+* Hardware Availability: Colorlice PCB
```

Comment:
This should link to some place where a user can get the hardware (a storefront listing or a GitHub repo).